src/compiler/commandLineParser.ts

```diff
@@ -417,6 +423,17 @@ namespace ts {
             category: Diagnostics.Module_Resolution_Options,
             description: Diagnostics.Type_declaration_files_to_be_included_in_compilation
         },
+        {

```

Comment:
i think this should be one level higher. it is similar to `includes`, `excludes` and `files` more than a compiler option.

Reply:
one other reason why this should not be here is do not think we want `tsc --references a\tsconfig.json` to work.

Reply:
✔️